package.json

```diff
@@ -51,6 +51,12 @@
       "apps/*",
       "packages/**",
       "scripts"
+    ],
+    "nohoist": [
+      "@fluentui-react-native/tester-win32/appium-windows-driver",
+      "@fluentui-react-native/tester/appium-mac2-driver",
+      "@fluentui-react-native/tester/appium-xcuitest-driver",
+      "@fluentui-react-native/tester/appium-windows-driver"
     ]
```

Comment:
dumb question, could we specify this in `apps/fluent-tester/package.json` instead? Not sure that's actually better but was curious.

Reply:
If nohoist is a property of workspaces I'm guessing not, but quick glance at documentation doesn't say one way or the other

Reply:
From what I've seen, this is common practice. Rather than having nohoise in multiple different package.json, we have one at the root with all the workspaces